docs/product/stats/index.mdx

```diff
@@ -10,11 +10,87 @@ The "Usage" tab shows your organization’s event and attachments usage, while a
+
+Events and attachments discarded on the client side due to the sampling configuration in the SDK or other factors. You can see common reasons below:
+
+#### Filtered
+
+| Reason                                                                              | Meaning                                                                            |
+| ----------------------------------------------------------------------------------- | ---------------------------------------------------------------------------------- |
+| [Browser Extensions](/concepts/data-management/filtering/#browser-extension-errors) | Filtered by browser extension.                                                     |
+| Denied Name                                                                         | Filtered due to name being denied.                                                 |
+| Disabled Namespace                                                                  | Filtered due to the namespace being disabled.                                      |
+| Error Message                                                                       | Filtered based on specific error message.                                          |
+| Filtered Transactions                                                               | Filtered because it was a call to a filtered transaction.                          |
+| Ip Address                                                                          | Filtered by IP address.                                                            |
+| Invalid CSP                                                                         | Filtered due to an invalid Content Security Policy (CSP) policy.                   |
+| [Legacy Browsers](/concepts/data-management/filtering/#legacy-browser-filters)      | Filtered out certain legacy versions of browsers that are known to cause problems. |
+| Localhost                                                                           | Filtered due to localhost restriction.                                             |
+| Release Version                                                                     | Filtered by release name (version).                                                |
+| [Web Crawlers](/concepts/data-management/filtering/#web-crawler-errors)             | Identified as known web crawler.                                                   |
+| ? (Others)                                                                          | Filtered due to other generic filtering rules not specified above.                 |
+
+#### Dropped
+
+Events and attachments discarded by the system due to rate limits, quotas, spike protection, [size limits](/concepts/data-management/size-limits), or invalid data. You can see the common reasons below:
+
+| Reason                          | Meaning                                                                                                                 |
```

Comment:
I was uncertain about adding this table here since it can get quite lengthy. However, I didn't want to place it elsewhere, as that might make it feel disconnected from the usage stats section. Thoughts?

Reply:
I would not list those at all. Those outcome reasons are not stable and meant for internal use. Apart from that, they refer to internals of the system that are confusing to a user.

Reply:
I understand those outcome reasons are for internal use and may confuse users. However, if users can see this info, we should provide at least a brief explanation to ensure clarity. Don't you agree?

Reply:
Let's expose these reasons in a simplified and grouped way that makes it easier for users to comprehend. I will post an update here with mapped reasons and improved descriptions.

Reply:
Here is a list that groups all discard reasons so that they can be understood by users. I've also added more relatable descriptions that point at docs to make this more actionable. 

https://gist.github.com/jan-auer/2d29e2ee6334bbcf32514b12b42b31ea

As for the different kinds of outcomes, please show them separately in the following combination at least in charts:

- Accepted (0)
- Filtered (1)
- Rate limited (2) + Abuse (4) + Cardinality Limited (6) -- currently shown as "Dropped" in the UI
- Invalid (3)
- Client Discard (5)